ReactiveFeedback/Feedback.swift

```diff
@@ -4,7 +4,21 @@ import enum Result.NoError
 
 public struct Feedback<State, Event> {
     public let events: (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>
+    
+    /// Creates an arbitrary Feedback, by transforming a sequence of State to a sequence of Events that mutate the State
+    ///
+    /// - parameters:
+    ///    - events: A closure which transforms the Signal<State, NoError> to the Signal<Event, NoError>
+    public init(events: @escaping (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>) {
+        self.events = events
+    }
 
+    /// Creates Control a Feedback which will perform effects when `query` exists (not nil)
+    /// and is different from previous, otherwise, cancels previously performed effects.
+    ///
+    /// - parameters:
+    ///     - query: A closure which defines for which value perform the effect
```

Comment:
I don't understand what you mean. :(

Reply:
I guess this sentence can be improved :)

Imaging paginated list state machine - we want to load next page only when we have a new value of it.

```swift
enum State {
   case initial(list: [Item])
   case loading(list: [Item], nextPage: URL)
   case loaded(list: [Item])

   var query: URL? {
      switch self { ... }
   }
}
```